package.json

```diff
@@ -8,50 +8,50 @@
     "url": "https://gitlab.com/antora/antora-ui-default.git"
   },
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 17.0.0"
   },
   "browserslist": [
     "last 2 versions"
   ],
   "devDependencies": {
     "@asciidoctor/core": "~2.2",
-    "autoprefixer": "~9.7",
+    "autoprefixer": "~10.4",
     "browser-pack-flat": "~3.4",
-    "browserify": "~16.5",
-    "cssnano": "~4.1",
+    "browserify": "~17.0",
+    "cssnano": "~5.1",
     "eslint": "~6.8",
     "eslint-config-standard": "~14.1",
     "eslint-plugin-import": "~2.20",
     "eslint-plugin-node": "~11.1",
     "eslint-plugin-promise": "~4.2",
     "eslint-plugin-standard": "~4.0",
-    "fancy-log": "~1.3",
+    "fancy-log": "~2.0",
     "font-awesome": "^4.7.0",
-    "fs-extra": "~8.1",
+    "fs-extra": "~10.0",
     "gulp": "~4.0",
     "gulp-concat": "~2.6",
     "gulp-connect": "~5.7",
     "gulp-eslint": "~6.0",
-    "gulp-imagemin": "~6.2",
-    "gulp-postcss": "~8.0",
+    "gulp-imagemin": "~8.0",
+    "gulp-postcss": "~9.0",
     "gulp-stylelint": "~13.0",
     "gulp-uglify": "~3.0",
-    "gulp-vinyl-zip": "~2.2",
+    "gulp-vinyl-zip": "~2.5",
     "handlebars": "~4.7",
     "highlight.js": "9.18.3",
```

Comment:
staying with this version for now since newer versions break callout formatting (see  #4)